src/main/java/gov/usds/case_issues/model/DateRange.java

```diff
@@ -0,0 +1,34 @@
+package gov.usds.case_issues.model;
+
+import java.time.ZonedDateTime;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.PastOrPresent;
+
+/**
+ * A simple container object for time/date ranges.
+ */
+public class DateRange {
+
+	private ZonedDateTime _start;
+	private ZonedDateTime _end;
+
+	public DateRange(@PastOrPresent @NotNull ZonedDateTime startTime, @PastOrPresent ZonedDateTime endTime) {
```

Comment:
Why the `PastOrPresent` annotation used? From what I can tell it excludes the now instance. not sure why we want this especially since if endTime is null we set it to now.

Reply:
I _think_ `@PastOrPresent` allows for `now`— I definitely agree that we want to allow that, I just thought that I had!

Reply:
I think I just might be a bit confused by the annotation description https://docs.jboss.org/hibernate/stable/beanvalidation/api/javax/validation/constraints/PastOrPresent.html